interface/src/app/tribe_client/resource.js

```diff
@@ -1,12 +1,33 @@
 /*
- * The resources for fetching the OAuth2 token from the tribe_client url
+ * The factories for fetching resources from the tribe_client urls
  */
 angular.module("adage.tribe_client.resource", ['ngResource'])
-    .factory('User', ['$resource', function($resource) {
-        return $resource('/tribe_client/return_user', {}, {});
-    }])
 
-    .factory('TribeSettings', ['$resource', function($resource) {
-        return $resource('/tribe_client/get_settings', {}, {});
-    }])
+.factory('User', ['$resource', function($resource) {
+  return $resource('/tribe_client/return_user', {}, {});
+}])
+
+.factory('TribeSettings', ['$resource', function($resource) {
+  return $resource('/tribe_client/get_settings', {}, {});
+}])
+
+.factory('Genesets', ['$resource', function($resource) {
+  return $resource(
+    'https://tribe.greenelab.com/api/v1/geneset/:creator/:slug/', {
+      creator: '@creator',
+      slug: '@slug'
+    },
+
+    {
```

Comment:
You have two different styles here: above, at line 16, it says

`param1, {`

where `{` is the beginning of `param2`, and here you have

```
},

{
```

where `}` is the end of `param2` and `{` is the beginning of `param3`. Maybe ESLint is trying to enforce something here, but if so I think it's doing it wrong. The style should be consistent.